src/components/GameRoom.js

```diff
@@ -174,20 +171,42 @@ function GameRoom({ room, setInRoom, userName, host, gamesWon,_id }) {
           <option value="option2">Game Lobby</option>
         </select> */}
       </div>
-
       <div>
         <h1>You are in Room {room}</h1>
-        <h2>Current players are: {players.join('-')}</h2>
-        <button onClick={leaveRoom}>Leave Room</button>        
+        <hr></hr>
+        {/* Attempted mapping of players into list form */}
+        <h2>
+          <strong>Current players are:</strong>{" "}
+        </h2>
+        <ul>
+          {players.map((player) => (
+            <div key={player}>
+              <li
+                style={{
+                  marginLeft: "45%",
+                  textAlign: "left",
+                }}
+              >
+                - {player}
+              </li>
+            </div>
+          ))}
+        </ul>
+        {console.log("players", players)}
+        <button onClick={leaveRoom}>Leave Room</button>
+        <button onClick={disconnectRoom}>Disconnect</button>
```

Comment:
The button disconnect is not necessary any more, now we have the Logout btn (line 167 this same file)